fdbserver/GetEncryptCipherKeys.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * GetCipherKeys.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBSERVER_GETCIPHERKEYS_H
+#define FDBSERVER_GETCIPHERKEYS_H
+#pragma once
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include "fdbserver/ServerDBInfo.h"
+#include "flow/BlobCipher.h"
+
+extern Future<std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>>> getLatestEncryptCipherKeys(
```

Comment:
remove `extern` keyword? (from here and other places as they are functions defined in header file and linker would be happy without extern keyword as well)

Reply:
Removed. For one time I'm getting duplicate symbol with the function names. Seemingly the `pragma once` directive didn't work. I tried again and now the duplicate symbol errors are gone. No idea what was wrong.